### DISCUSSION TITLE
Form style customisation

### DISCUSSION THREAD

**Discussion by AbdelilahDerfoufi** [symfony/symfony#45292] [Q&A]
How can I customize the form style of the form generated by form type? 

Here is what it shows:

![image](https://user-images.githubusercontent.com/49759161/152195506-cbaa6114-71e9-4acf-bac7-ea94e044459d.png)

**Comment by jaures-kano** [comment]
hi, You want to customize bootstrap style or grid ?

**Reply by AbdelilahDerfoufi** [reply]
I added bootstrap5_layout.html.twig line in twig.yml but it's too wide, what can I do more to customize it?

**Reply by AbdelilahDerfoufi** [reply]
Style

**Comment by chapterjason** [comment]
Have you already looked in the [documentation](https://symfony.com/doc/current/form/form_customization.html)?

**Reply by AbdelilahDerfoufi** [reply]
Yes.
